device/arista/x86_64-arista_7060x6_16pe_384c/Arista-7060X6-16PE-384C-O128S2-COPPER-LAB/port_config.ini

```diff
@@ -55,18 +55,18 @@ Ethernet208  21,22,23,24      etp18i  18     400000  rs
 Ethernet212  17,18,19,20      etp18j  18     400000  rs
 Ethernet216  1,2,3,4          etp18k  18     400000  rs
 Ethernet220  5,6,7,8          etp18l  18     400000  rs
-Ethernet224  509,510,511,512  etp19a  19     400000  rs
-Ethernet228  505,506,507,508  etp19b  19     400000  rs
-Ethernet232  489,490,491,492  etp19c  19     400000  rs
-Ethernet236  493,494,495,496  etp19d  19     400000  rs
-Ethernet240  485,486,487,488  etp19e  19     400000  rs
-Ethernet244  481,482,483,484  etp19f  19     400000  rs
-Ethernet248  477,478,479,480  etp19g  19     400000  rs
-Ethernet252  473,474,475,476  etp19h  19     400000  rs
-Ethernet256  465,466,467,468  etp19i  19     400000  rs
-Ethernet260  469,470,471,472  etp19j  19     400000  rs
-Ethernet264  457,458,459,460  etp19k  19     400000  rs
-Ethernet268  461,462,463,464  etp19l  19     400000  rs
+Ethernet224  509,510,511,512  etp21a  21     400000  rs
```

Comment:
after the update, this ethernet port sequence looks a bit weird. @prgeor are you expecting this?

Reply:
@r12f looks like the EthernetXX mapping to etpXX is not right but the etpX to index mapping is as per sonic convention. I will check this internally and get back if we can live with this.